### PR TITLE
refactor:add new ccd case document am api

### PR DESCRIPTION
**Pull request by dharmendrak**

### CODE REVIEW

File: compose/backend.yml [hunk 1]
```diff
@@ -70,12 +70,14 @@ services:
       DATA_STORE_DB_USE_SSL: "${DB_USE_SSL}"
       DATA_STORE_IDAM_KEY: "${IDAM_KEY_CCD_DATA_STORE}"
       DATA_STORE_TOKEN_SECRET: iuasbcuasdcbasdgcasdgcuysachjsacyasdgjcgasdj
-      DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_gw,ccd_gateway,civil_service,ccd_data,ccd_ps,xui_webapp,aac_manage_case_assignment
+      DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_gw,ccd_gateway,civil_service,fpl_case_service,ccd_data,ccd_ps,aac_manage_case_assignment,xui_webapp,ccd_case_document_am_api,am_role_assignment_service
```
Comment (ckalista) [inline compose/backend.yml:73]: Why do we have a reference to `fpl`?

File: compose/dm-store.yml [hunk 2]
```diff
@@ -39,7 +39,7 @@ services:
       LOGGING_LEVEL_UK_GOV_HMCTS_IDAM: DEBUG
       IDAM_USER_BASE_URI: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
       IDAM_S2S_BASE_URI: http://service-auth-provider-api:8080
-      S2S_NAMES_WHITELIST: sscs,divorce,ccd,em_gw,document-management-store,idam-api,xui_webapp,ccd_gw,civil_service
+      S2S_NAMES_WHITELIST: sscs,divorce,ccd,em_gw,document-management-store,ccd_gw,idam-api,fpl_case_service,xui_webapp,ccd_case_document_am_api,civil_service
```
Comment (ckalista) [inline compose/dm-store.yml:42]: Again, why this reference to `fpl` here?

Comment (dharmendrak) [reply compose/dm-store.yml:42]: removing them

File: compose/backend.yml (hunk 1 above)
Comment (dharmendrak) [reply compose/backend.yml:73]: removing them

**Pull request merged**